data/moves.ts

```diff
@@ -18312,7 +18312,8 @@ export const Moves: {[moveid: string]: MoveData} = {
 		volatileStatus: 'torment',
 		condition: {
 			noCopy: true,
-			onStart(pokemon) {
+			onStart(pokemon, source, effect) {
+				if (effect?.id === 'gmaxmeltdown') this.effectState.duration = 3;
```

Comment:
This could go after after the dynamax check perhaps?

Reply:
It all works the same bye I guess I can, I’ll do it when I wake up